.github/workflows/run_merge_test.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+name: Running tests
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  test_examples:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Run the test examples
+        shell: bash -l {0}
+        run: |
+          pip install seaborn=0.12, mpi4py, pytorch, scikit-learn, rasterio, scikit-image
```

Comment:
I think probably you would want to set up a conda env so it works the same as our other tests (otherwise we end up maintaining two different types of workflows). miniconda install should also be more uptodate and stable I hope

See here https://github.com/aidotse/PASEOS/blob/a8126c7b231088d27822cfb386b3f617711bd21c/.github/workflows/run_tests.yml#L40

Reply:
Yes, using the same environment as the tests now.